app/src/interfaces/many-to-one/options.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+	<div class="grid">
+		<v-notice type="warning" v-if="collection == null">
+			{{ $t('interfaces.one-to-many.no_collection') }}
+		</v-notice>
+		<div v-else class="full">
+			<p class="type-label">{{ $t('interfaces.many-to-one.display_template') }}</p>
+			<v-field-template :collection="collection" v-model="template" :depth="0"></v-field-template>
```

Comment:
The m2o support dot notation fields, so lets allow them to go a level deeper here

```suggestion
			<v-field-template :collection="collection" v-model="template" :depth="1" />
```